third-party/qthread/qthread-1.10/src/interfaces/chapel/tasks-qthreads.c

```diff
@@ -615,13 +618,125 @@ void chpl_task_exit(void)
 #endif /* QTHREAD_MULTINODE */
 }
 
+//
+// Tasking callback support.
```

Comment:
We could share install and uninstall among all the tasking layers, but when the callbacks are called the info passed to them has to come from implementation-specific places in each tasking layer, so at least some of that has to be implementation-specific. It's an attractive idea, though, and I'll pursue this before I do the next one.

I don't think moving the qthreads tasking shim into our runtime would help with this specifically, though.